src/Tests/CreateOrderTest.php

```diff
@@ -0,0 +1,131 @@
+<?php
+
+namespace GingerPluginSdk\Tests;
+
+use GingerPluginSdk\Collections\AdditionalAddresses;
+use GingerPluginSdk\Collections\OrderLines;
+use GingerPluginSdk\Collections\PhoneNumbers;
+use GingerPluginSdk\Collections\Transactions;
+use GingerPluginSdk\Entities\Address;
+use GingerPluginSdk\Entities\Customer;
+use GingerPluginSdk\Entities\Extra;
+use GingerPluginSdk\Entities\Line;
+use GingerPluginSdk\Entities\Order;
+use GingerPluginSdk\Entities\PaymentMethodDetails;
+use GingerPluginSdk\Entities\Transaction;
+use GingerPluginSdk\Properties\Country;
+use GingerPluginSdk\Properties\Currency;
+use GingerPluginSdk\Properties\Email;
+use GingerPluginSdk\Properties\Locale;
+use PHPUnit\Framework\TestCase;
+
+class CreateOrderTest extends TestCase
+{
+    private Order $order;
+
+    public function setUp(): void
+    {
+        $_SERVER["REMOTE_ADDR"] = "173.0.2.5";
+        $_SERVER["HTTP_USER_AGENT"] = "PHPUnit Tests";
+
+        $this->client = new \GingerPluginSdk\Client(
+            new \GingerPluginSdk\Properties\ClientOptions(
+                endpoint: "https://api.online.emspay.eu",
+                useBundle: true,
+                apiKey: getenv('GINGER_API_KEY'))
```

Comment:
Just question. For now it is not necessary, but is it possible in future make variable endpoint like apiKey? Then this tests could be used with different banks and their endpoints